src/components/InfoWindow/Contents/BookMark.tsx

```diff
@@ -1,20 +1,48 @@
-import React from 'react';
 import { toast } from 'react-hot-toast';
 import { BookmarkFilledIcon, BookmarkIcon } from '@assets/index';
 import styled from '@emotion/styled';
+import { MessageToFlutterType } from '@constants/flutterCallback';
+import messageToFlutter from '@libs/webview/messageToFlutter';
+import { useRecoilValue } from 'recoil';
+import { activatedCafeIdAtom } from '@states/infoWindow';
+import { favoriteListAtom } from '@states/favoriteList';
+import { isWebViewAtom } from '@states/isWebView';
+import { useEffect, useRef } from 'react';
 
-interface BookMarkProps {
-  isBookmarked: boolean;
-}
+function BookMark() {
+  const activatedCafeId = useRecoilValue<number | null>(activatedCafeIdAtom);
+  const favoriteList = useRecoilValue(favoriteListAtom);
+  const isBookmarked = activatedCafeId !== null ? favoriteList.includes(Number(activatedCafeId)) : false;
+  const isWebView = useRecoilValue<boolean>(isWebViewAtom);
+  const isMounted = useRef(false);
 
-function BookMark({ isBookmarked }: BookMarkProps) {
   const onBookmarkClick = () => {
-    toast.success(isBookmarked === true ? '북마크가 해제되었습니다.' : '북마크가 추가되었습니다.');
+    if (isWebView) {
+      if (isBookmarked) {
+        messageToFlutter(MessageToFlutterType?.removeFavorite, activatedCafeId);
+      } else {
+        messageToFlutter(MessageToFlutterType.addFavorite, activatedCafeId);
```

Comment:
(마이너)

삼항연산자를 쓰면
```suggestion
      const bookMarkActionType = isBookMarked ?  MessageToFlutterType.removeFavorite : MessageToFlutterType.addFavorite;

        messageToFlutter(bookMarkActionType, activatedCafeId);
```
이런식으로 코드를 줄일수 있을거 같은데 한번 말씀드려봅니다!

Reply:
두 커밋 제안 모두 적용하였습니다 감사합니다